ExtraDry/ExtraDry.Blazor/Components/Dry/DryStaticFilter.razor

```diff
@@ -0,0 +1,5 @@
+﻿@namespace ExtraDry.Blazor
+@typeparam TItem
+
+<div class="@CssClasses" @attributes="@UnmatchedAttributes">
```

Comment:
Does this need a visualization?  I don't think there's much of purpose.  Perhaps for developer debugging in the browser? If so, then perhaps `<div style="display: none" data-text-filter="some filter" data-initial-level="2" />`.

Reply:
Updated as it doesn't need a visualization